### DISCUSSION TITLE
How to ignore escaping completely

### DISCUSSION THREAD

**Discussion by pvamshi** [syntax-tree/unist#44] [Q&A] [answered]
I am using `remark-stringify`. It escapes lot of characters by default. How do I make it not escape any character. I tried passing `unsafe: []`  , but it didnt work. Without going to details of implementation is there an easier way to handle this ? 

For ex. 
```html
<li> [.] text </li>
```
gets converted to 
```md
 - \[.] text
 ```

But I prefer 
```md
- [.] text
```

So are `<li> #hashtag</li>` , `<li> +element </li>`

**Comment by wooorm** [comment, marked as answer]
No. Escaping is essential.

**Reply by pvamshi** [reply]
Not always. But I understand if you don't want to provide an option for it. I will figure out an alternative way. 👍

**Reply by wooorm** [reply]
> Not always.

Not every escape is required, sure. But you wanted to turn all escapes off, and you shouldn’t: escaping in general is required.

`[.]` is escaped because otherwise, if you add `[.]: #` somewhere, it’d turn into a link.

If you can find ways to make `#` or `+` smarter, you can PR a fix to [`unsafe.js`](https://github.com/syntax-tree/mdast-util-to-markdown/blob/f3df7410049ed426ef8734ec762a38aa2feee73f/lib/unsafe.js#L38) or suggest a change in that project

**Reply by aandis** [reply]
hi @wooorm is there a way to escape `[ ]` so that its outputted as is from ast to markdown?

**Reply by wooorm** [reply]
Please clarify your question. This is way too vague.
